README.md

```diff
@@ -62,9 +79,54 @@ Alternatively, create a new config file named `config.json`,
             "commit": "6bd6b76d1156e20e45d1016f355d154141c7e5b9",
             "relative_root": "contracts"
         }
+    },
+    "raise_exception": true,
+    "bytecode_comparison": {
+        "hardhat_config_name": "holesky_hardhat.config.js",
+        "constructor_calldata": {
+            "0x28FAB2059C713A7F9D8c86Db49f9bb0e96Af1ef8": "000000000000000000000000ab89ed3d8f31bcf8bb7de53f02084d1e6f043d34000000000000000000000000e92329ec7ddb11d25e25b3c21eebf11f15eb325d00000000000000000000000000000000000000000000000000000000000000600000000000000000000000000000000000000000000000000000000000000000"
+        },
+        "constructor_args": {
+            "0x28FAB2059C713A7F9D8c86Db49f9bb0e96Af1ef8": [
+                "0xab89ED3D8f31bcF8BB7De53F02084d1e6F043D34",
+                "0xE92329EC7ddB11D25e25b3c21eeBf11f15eB325d",
+                ""
```

Comment:
Is it a good example when we have both args and calldata for the same contract address? If so, what would be the order of preference?

In my view, the program should treat this config being ambiguous and invalid (malformed) :warning:

Reply:
i've prefered prepared calldata from field 'constructor_calldata' to raw calldata in 'constructor_args'. Fixed example

Reply:
not sure if it's a good approach still: it's too ambiguous and undocumented, why we can't enforce a uniqueness?